### DISCUSSION TITLE
Cannot open any github page with Firefox

### DISCUSSION THREAD

**Discussion by hard-sign** [community/community#86050] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

A github page cannot be open with Firefox.
All the pages load successfully from the same machine with Google Chrome, Epiphany and curl.

User agent: Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:121.0) Gecko/20100101 Firefox/121.0
IP: 185.32.135.164
Connecting to internet via provider’s NAT; no VPN, no proxies.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by LuksJobs** [comment]
Clear Cache and Cookies:

- Go to Preferences (or Options) in Firefox.
- Navigate to Privacy & Security.
- Under Cookies and Site Data, click Clear Data.
- Select Cookies and Site Data and Cached Web Content, then click Clear.

If none of the above solutions work, consider a clean reinstall of Firefox.

**Reply by hard-sign** [reply]
All of this is done, and nothing helps.
Any other site work fine.
I’ve found the closed issue: https://github.com/microsoft/playwright/issues/20749
My problem is the same. Despite the issue is closed, the problem is not fixed.

**Reply by LuksJobs** [reply]
Try changing your DNS settings to use a different DNS server. You can use public DNS servers like Google's (8.8.8.8 and 8.8.4.4) or Cloudflare's (1.1.1.1).

**Reply by hard-sign** [reply]
exactly the same:
```
➜ nslookup github.com
Server:		127.0.0.53
Address:	127.0.0.53#53

Non-authoritative answer:
Name:	github.com
Address: 140.82.121.3

➜ nslookup github.com 8.8.8.8
Server:		8.8.8.8
Address:	8.8.8.8#53

Non-authoritative answer:
Name:	github.com
Address: 140.82.121.4
```
I see `NS_BINDING_ABORTED` in Firefox developer console. It looks exactly as the old bug desctibed in the linked issue.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @hard-sign ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Projects and Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Reply by kimallen** [reply]
Including a link to where that is would be super helpful here.

**Comment by hard-sign** [comment]
I tried to open Github right now from another internet provider. Everything works fine.
It looks like Github blocks a query with combination of browser identification string and IP address.

**Comment by dariohsb** [comment]
Clear your history including cache and cookies and try again

**Comment by sashabeep** [comment, marked as answer]
@hard-sign Have you tried to set **security.tls.version.enable-deprecated** to false in about:config ? see https://bugzilla.mozilla.org/show_bug.cgi?id=1872383

**Reply by hard-sign** [reply]
This solution works. Thank you!
